packages/react-cards/src/components/Card/examples/Card.Vertical.Example.tsx

```diff
@@ -99,13 +99,13 @@ export class CardVerticalExample extends React.Component<{}, {}> {
 
     return (
       <Stack horizontal tokens={sectionStackTokens}>
-        <Card>
+        <Card aria-label="Basic vertical card">
           <Card.Item>
             <Text>Basic vertical card</Text>
           </Card.Item>
         </Card>
 
-        <Card onClick={alertClicked} tokens={cardTokens}>
+        <Card aria-label="Clickable vertical card with image bleeding at the center of the card" onClick={alertClicked} tokens={cardTokens}>
```

Comment:
In a real world example you probably would include some information from the card in the aria-label. Otherwise this description would be the same for every card, and the user would have to navigate inside the card with scan-mode in order to get any information that is in the card.

Reply:
That's true, however, because that's the concern of the developer using the controllers. Since these examples are meant to exemplify controllers with dummy data I don't think there is any issue with the `aria-label` being set as is.